HeterogeneousCore/CUDAServices/test/test_main.cpp

```diff
@@ -0,0 +1,2 @@
+#define CATCH_CONFIG_MAIN
+#include "catch.hpp"
```

Comment:
Why both `test_main.cc` and `test_main.cpp` which are internally identical?

Reply:
`test_main.cc` seems to be completely unused, I wonder how that creeped in...